Automattic/wp-calypso
# Performance Profiler: Update loading image while getting LLM suggestions

## Proposed Changes

* Update loading icon with big sky rotating icon. 

## Why are these changes being made?


* As per the suggestion received in https://github.com/Automattic/wp-calypso/pull/94254#discussion_r1746681775

## Testing Instructions



* Visit following URL on calypso live link. 
```
/speed-test-tool?url=https%3A%2F%2Fwww.atliq.com%2F&hash=eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzI1NiJ9.eyJpZCI6NjkzNH0.uoPBbETEfzOEqTdJ_kro-zUpmvSF6vQz7JabJYumwME
````
- Click on any recommendation and notice the loading icon animation. 

https://github.com/user-attachments/assets/ca43e45a-7157-47af-b092-99a18c7f3319



## Pre-merge Checklist
